src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -101,7 +101,8 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
 
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
+        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress,
+                personToEdit.getPostalCode(), updatedTags);
```

Comment:
`personToEdit.getPostalCode()` this would return the original postal code instead of the new value.

But isn't this edit command? 😅😅 I don't think you have to touch this.

Reply:
Yeap, I set it to be like this because I have not officially change the edit function yet for postal code. I think if I did not make any changes to the edit, the test cases for the edit will not pass, and hence the build will not go through at the GitHub. The current edit implementation is just temporary, I will edit this when I doing the next iteration. But yes, thanks for noticing and bringing it up :)